src/applications/vaos/referral-appointments/ChooseDateAndTime.jsx

```diff
@@ -240,14 +253,21 @@ export const ChooseDateAndTime = () => {
               </a>
             </div>
           </address>
-          <p>Phone: {provider.orgPhone}</p>
+          <p>
+            Phone:{' '}
+            <va-telephone
+              contact={provider.orgPhone}
+              data-testid="provider-telephone"
+            />
+          </p>
           <p>
             {provider.driveTime} ({provider.driveDistance})
           </p>
           <h2>Choose a date and time</h2>
           <p>
             Select an available date and time from the calendar below.
-            Appointment times are displayed in {`${tzLong} (${tzShort})`}.
+            Appointment times are displayed in{' '}
+            {`${getTimezoneDescByFacilityId(referral.provider)}`}.
```

Comment:
this will come from referring facility...but not important here, and can get updated in the ticket that adds the mock to this page.